### PR TITLE
Lvrubym 243 : change styles

### PR DESCRIPTION
**Pull request by HelenMaherovska**
## JIRA

* [Main JIRA ticket](http://ssu-jira.softserveinc.com/browse/LVRUBYM-220)
* [JIRA subtask](http://ssu-jira.softserveinc.com/browse/LVRUBYM-317)

## Code reviewers

- [ ] @vikabelous 
- [ ] @skrypnyk88 
- [ ] @Mykhailo123 

### Second Level Review

- [ ] @4asof4ik

## Summary of issue

Need to implement CI

## Summary of change

- implement CI(JSCS)

## Testing approach

Test with JSCS

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=90%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Comment by skrypnyk88** [comment]
Need to implement CI :) Please change this.

**Comment by skrypnyk88** [comment]
Please rename last 3 commits.

File: app/img/Out/.directory [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[Dolphin]
```
Comment (skrypnyk88) [inline app/img/Out/.directory]: What is this the file?

**Comment by skrypnyk88** [comment]
Why do you use logo.gif and Logo.png?

File: app/js/main/components/main.template.html [hunk 2]
```diff
@@ -1,74 +1,107 @@
...
+          <md-option value="class a">Class A</md-option>
+          <md-option value="class b">Class B</md-option>
+          <md-option value="class c">Class C</md-option>
+          <md-option value="class d">Class D</md-option>
+        </md-select>
 
-    <div flex layout="row" layout-align="space-around center">
-
-      <md-select ng-model="$ctrl.currentGroupDay.group_id" >
+<!--       <md-select ng-model="$ctrl.currentGroupDay.group_id" >
```
Comment (skrypnyk88) [inline app/js/main/components/main.template.html]: Please delete this comment lines if we don't need them.

File: app/js/main/components/main.template.html [hunk 3]
```diff
@@ -1,74 +1,107 @@
...
-          <li><a ui-sref="#">Health</a></li>
-          <li><a ui-sref="#">Bottle</a></li>
-          <li><a ui-sref="#">Meals</a></li>
-          <li><a ui-sref="#">Potty</a></li>
+      <li class="teacher">
+        <a ui-sref="#">
+          Teacher report
+        </a>
+      </li>
+      <!-- <li> -->
```
Comment (skrypnyk88) [inline app/js/main/components/main.template.html]: Delete this line.

File: app/js/main/components/main.template.html [hunk 4]
```diff
@@ -1,74 +1,107 @@
...
+        <ul class="reports1">
+          <li>
+            <a ui-sref="main.presence">
+              Presence
+            </a>
+          </li>
+          <li>
+            <a ui-sref="main.presence">
```
Comment (skrypnyk88) [inline app/js/main/components/main.template.html]: Why ui-sref="main.presence"

File: app/js/main/components/main.template.html [hunk 5]
```diff
@@ -1,74 +1,107 @@
...
+            <a ui-sref="#">
+              Meals
+            </a>
+          </li>
+          <li>
+            <a ui-sref="">
+              Potty
+            </a>
+          </li>
+          <!-- <li><a ui-sref="main.potty_report">Potty</a></li> -->
```
Comment (skrypnyk88) [inline app/js/main/components/main.template.html]: Please delete this line

File: app/js/main/components/main.template.html [hunk 6]
```diff
@@ -1,74 +1,107 @@
...
           <li><a ui-sref="#">Nap</a></li>
-          <li><a ui-sref="#">Achivements</a></li>
-          <li><a ui-sref="#">Supplies</a></li>
-          <li><a ui-sref="#">Our Day</a></li>
-          <li><a ui-sref="#">My Day</a></li>
+          <li><a ui-sref="#">Rewards</a></li>
+          <li><a ui-sref="#">Personal notes</a></li>
+          <li><a ui-sref="#">Class notes</a></li>
+          <!-- <li><a ui-sref="#">My Day</a></li> -->
```
Comment (skrypnyk88) [inline app/js/main/components/main.template.html]: The same

File: app/js/main/components/main.template.html [hunk 7]
```diff
@@ -1,74 +1,107 @@
...
+        <ul class="reports2">
+          <li><a href="">Parent Sharings</a></li>
+          <li><a href="">Messages</a></li>
+          <li><a href="">Notifications</a></li>
+          <li><a href="">Calendar</a></li>
+          <li><a href="">Photo Gallery</a></li>
+          <li><a href="">Rewards</a></li>
+          <li><a href="">Contacts</a></li>
+        </ul>
+      <!-- </li> -->
```
Comment (skrypnyk88) [inline app/js/main/components/main.template.html]: The same

File: app/js/main/main.module.js [hunk 8]
```diff
@@ -4,6 +4,8 @@ var loginModule = require('./../login/login.module.js');
 var presenceReportModule = require('./../presenceReport/presenceReport.module.js');
 var studentModule = require('./../student/student.module.js');
 var groupService = require('./../common/services/group.service.js');
+var pottyReportModule = require('./../pottyReport/pottyReport.module.js');
```
Comment (skrypnyk88) [inline app/js/main/main.module.js]: The same

File: app/js/main/main.module.js [hunk 9]
```diff
@@ -14,7 +16,9 @@ module.exports = angular
     loginModule.name,
     presenceReportModule.name,
     studentModule.name,
-    groupService.name
+    groupService.name,
+    pottyReportModule.name,
```
Comment (skrypnyk88) [inline app/js/main/main.module.js]: The same

File: app/js/pottyReport/components/pottyReport.component.js [hunk 10]
```diff
@@ -0,0 +1,10 @@
+module.exports = angular
```
Comment (skrypnyk88) [inline app/js/pottyReport/components/pottyReport.component.js]: Please delete this component

File: app/js/pottyReport/components/pottyReport.template.html [hunk 11]
```diff
@@ -0,0 +1,7 @@
+<div class="btn_container" layout="column">
```
Comment (skrypnyk88) [inline app/js/pottyReport/components/pottyReport.template.html]: The same

File: app/js/pottyReport/pottyReport.module.js [hunk 12]
```diff
@@ -0,0 +1,8 @@
+var pottyReportComponent = require('./components/pottyReport.component.js');
```
Comment (skrypnyk88) [inline app/js/pottyReport/pottyReport.module.js]: The same

File: app/js/pottyReport/pottyReport.route.js [hunk 13]
```diff
@@ -0,0 +1,9 @@
+module.exports = angular
```
Comment (skrypnyk88) [inline app/js/pottyReport/pottyReport.route.js]: The same

File: app/js/static/components/static.template.html [hunk 14]
```diff
@@ -0,0 +1,50 @@
...
+			<div class="name" layout="column" layout-align="center">
+				<span class="first-name">Andriy</span>
+				<span class="last-name">Kalyta</span>
+			</div>
+		</div>
+		</md-list-item>
+	</md-list>
+</div>
+
+		<div class="presence-report">
```
Comment (skrypnyk88) [inline app/js/static/components/static.template.html]: Why presence-report

File: app/js/static/components/static.template.html [hunk 15]
```diff
@@ -0,0 +1,50 @@
...
+					<section class="student-btn" layout="column" layout-align="center center">
+					<div class="btn-container" layout="row">
+						<md-button class="group-btn present-btn">Present</md-button>
+						<md-button class="group-btn absent-btn">Absent</md-button>
```
Comment (skrypnyk88) [inline app/js/static/components/static.template.html]: The same

File: app/js/static/components/static.template.html [hunk 16]
```diff
@@ -0,0 +1,50 @@
...
+					</div>
+					<div class="input-container" layout="row" layout-align="space-around">
+			        <input class="input-time" type="text" name="hours" value="13:15">
+			        <input class="input-time" type="text" name="minutes" value="18:45">
+			    </div>
+				</section>
+		</div>
+
+
+	<md-select flex="5" ng-model="" placeholder="30 ml" class="blue-select md-no-underline" layout="row" layout-align="end end">
```
Comment (skrypnyk88) [inline app/js/static/components/static.template.html]: Very long line

**Comment by skrypnyk88** [comment]
app/css/fonts/MyriadPro-Regular.otf
How will we use the file?

File: app/js/main/components/main.template.html (hunk 2 above)
Comment (HelenMaherovska) [inline app/js/main/components/main.template.html]: ok,I'll do that

File: app/js/main/components/main.template.html (hunk 4 above)
Comment (HelenMaherovska) [inline app/js/main/components/main.template.html]: I have to delete it

File: app/js/main/components/main.template.html [hunk 17]
```diff
@@ -1,74 +1,107 @@
-<div class="header">
-  <md-toolbar id="toolbar" class="md-menu-toolbar" layout="row">
+<header class="header" layout="row" layout-align="space-between center">
     <div class="logo">
-      <img  src="app/img/profile/logo.jpg" alt="logo">
+      <img class="logo" src="app/img/profile/logo.gif" alt="logo">
```
Comment (vikabelous) [inline app/js/main/components/main.template.html]: class `logo` inside class `logo` ?

File: app/js/main/components/main.template.html [hunk 18]
```diff
@@ -1,74 +1,107 @@
...
     </div>
+        <md-select flex="8" ng-model="classes" placeholder="Class A" class="group-select md-no-underline">
```
Comment (vikabelous) [inline app/js/main/components/main.template.html]: We already had working select with groups. The one you commented below. Please fix.

File: app/js/main/components/main.template.html (hunk 2 above)
Comment (vikabelous) [inline app/js/main/components/main.template.html]: Do not delete this! This is working select with groups.

File: app/js/main/components/main.template.html [hunk 19]
```diff
@@ -1,74 +1,107 @@
...
+    <span class="username">Jesica Alba</span>
+      <md-menu md-position-mode="target-right bottom">
+      <md-icon ng-click="$mdOpenMenu()">expand_more</md-icon>
+      <md-menu-content width="3">
+      <md-menu-item>
+        <md-button>Profile</md-button>
+      </md-menu-item>
+      <md-menu-divider role="separator"></md-menu-divider>
+      <md-menu-item>
+        <md-button>Sign out</md-button>
```
Comment (vikabelous) [inline app/js/main/components/main.template.html]: We had logout action on this button. 
```
 <md-button ng-click="$ctrl.logout()">Sign out</md-button>
```

File: app/js/main/components/main.template.html [hunk 20]
```diff
@@ -1,74 +1,107 @@
...
-          <li><a ui-sref="#">Potty</a></li>
+      <li class="teacher">
+        <a ui-sref="#">
+          Teacher report
+        </a>
+      </li>
+      <!-- <li> -->
+        <ul class="reports1">
+          <li>
+            <a ui-sref="main.presence">
```
Comment (vikabelous) [inline app/js/main/components/main.template.html]: `main.presenceReport`

File: app/js/static/components/static.template.html (hunk 14 above)
Comment (vikabelous) [inline app/js/static/components/static.template.html]: It should be general class for all reports, just name it `report`  or `report-container`

File: app/img/Out/Bottle.svg [hunk 21]
```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
```
Comment (okohuch) [inline app/img/Out/Bottle.svg]: Could you explain why do you use `svg` format for the images????

File: app/js/main/components/main.component.js [hunk 22]
```diff
@@ -33,4 +33,3 @@ function MainController($scope, $state, auth, currentGroupDay) {
     function() { $state.reload($state.current); }
   );
```
Comment (okohuch) [inline app/js/main/components/main.component.js]: Delete the changes of this file from the PR

File: app/js/static/components/static.template.html [hunk 23]
```diff
@@ -0,0 +1,50 @@
+<div class="student">
+	<md-list>
```
Comment (okohuch) [inline app/js/static/components/static.template.html]: What about alignment in this file?

**Pull request closed without merging**